doc/src/docs/InputOutputReference/01d-InputOutputReference.md

```diff
@@ -4278,6 +4278,239 @@ This is the total heating output of the DX coil in Joules over the time step bei
+
+#### Field: Availability Schedule Name
+
+This alpha field defines the name of the coil availability schedule. A name should be entered to define the availability of the coil. Schedule values of 0 denote the DX cooling coil is off. A schedule value greater than 0 indicates that the coil can operate during the time period. If this field is blank, the schedule has values of 1 for all time periods.
+
+#### Field: Coil Air Inlet Node
+
+This alpha field defines the name of the air inlet node entering the DX cooling coil.
+
+#### Field: Coil Air Outlet Node
+
+This alpha field defines the name of the air outlet node exiting the DX cooling coil.
+
+#### Field: Rated Total Cooling Capacity
+
+This numeric field defines the gross rated total cooling capacity of the DX cooling coil in watts. The total cooling capacity should be a gross , i.e., the effect of supply air fan heat NOT accounted for.
+ 
+#### Field: Rated Sensible Heat Ratio
+
+This numeric field defines the gross sensible heat ratio (sensible capacity divided by total cooling capacity) of the DX cooling coil at rated conditions. Both the sensible and total cooling capacities used to define the Rated SHR should be gross (i.e., the effect of supply air fan heat is NOT accounted for)
+	   
+#### Field: Indoor Unit Reference Superheating 
+
+This numeric field defines the reference superheating degrees of the indoor unit. If this field is blank, the default value of 5.0 C is used. 
+	   
+#### Field: Indoor Unit Evaporating Temperature Function of Superheating Curve Name    
+
+This alpha field defines the name of a quadratic performance curve that parameterizes the variation of indoor unit evaporating temperature as a function of superheating degrees. The output of this curve is the temperature difference between the coil surface air temperature and the evaporating temperature.
+   
+#### Field: Name of Water Storage Tank for Condensate Collection
+
+This field is optional. It is used to describe where condensate from the coil is collected. If blank or omitted, then any coil condensate is discarded. Enter the name of Water Storage Tank object defined elsewhere and the condensate will then be collected in that tank.
+
+Following is an example input for a Coil:Cooling:DX:VariableRefrigerantFlow:FluidTemperatureControl object.
+
+```idf
+  Coil:Cooling:DX:VariableRefrigerantFlow:FluidTemperatureControl,
+    TU1 VRF DX Cooling Coil, !- Name
+    VRFAvailSched,           !- Availability Schedule Name
+    TU1 VRF DX CCoil Inlet Node,  !- Coil Air Inlet Node
+    TU1 VRF DX CCoil Outlet Node,  !- Coil Air Outlet Node
+    autosize,                !- Rated Total Cooling Capacity {W}
+    autosize,                !- Rated Sensible Heat Ratio
+	3,                       !- Indoor Unit Reference Superheating Degrees {C}    
+	IUEvapTempCurve,         !- Indoor Unit Evaporating Temperature Function of Superheating Curve Name    
```

Comment:
Not sure in Markdown. I think you can use tabs as long as it looks good.

On 8/27/2015 3:25 PM, Rongpeng Zhang wrote:
>
> In doc/src/docs/InputOutputReference/01d-InputOutputReference.md 
> <https://github.com/NREL/EnergyPlus/pull/5026#discussion_r38135973>:
>
> > +#### Field: Name of Water Storage Tank for Condensate Collection
> > +
> > +This field is optional. It is used to describe where condensate from the coil is collected. If blank or omitted, then any coil condensate is discarded. Enter the name of Water Storage Tank object defined elsewhere and the condensate will then be collected in that tank.
> > +
> > +Following is an example input for a Coil:Cooling:DX:VariableRefrigerantFlow:FluidTemperatureControl object.
> > +
> > +```idf
> > +  Coil:Cooling:DX:VariableRefrigerantFlow:FluidTemperatureControl,
> > +    TU1 VRF DX Cooling Coil, !- Name
> > +    VRFAvailSched,           !- Availability Schedule Name
> > +    TU1 VRF DX CCoil Inlet Node,  !- Coil Air Inlet Node
> > +    TU1 VRF DX CCoil Outlet Node,  !- Coil Air Outlet Node
> > +    autosize,                !- Rated Total Cooling Capacity {W}
> > +    autosize,                !- Rated Sensible Heat Ratio
> > +	3,                       !- Indoor Unit Reference Superheating Degrees {C}
> > +	IUEvapTempCurve,         !- Indoor Unit Evaporating Temperature Function of Superheating Curve Name
>
> Tab character again... I will fix that.
>
> Just to confirm, the new EPlus custom is to use space instead of tab, 
> right?
>
> —
> Reply to this email directly or view it on GitHub 
> <https://github.com/NREL/EnergyPlus/pull/5026/files#r38135973>.
>

-- 
Richard Raustad
Florida Solar Energy Center
1679 Clearlake Road
Cocoa, FL 32922
Ph: (321)638-1454
http://www.fsec.ucf.edu/en/

Program Director
Electric Vehicle Transportation Center
http://evtc.fsec.ucf.edu/